### DISCUSSION TITLE
Times showing when the event is "all day"

### DISCUSSION THREAD

**Discussion by dailysol** [liedekef/events-made-easy#589] [Q&A] [answered]
Is there a way to block the the times for an all day event?  Thank you for your help

**Comment by liedekef** [comment]
And where is it showing?
In the single event view, you can change the event layout to your liking.
In the event listing, use eme_if to check if the event is all day or not by checking the value of #_IS_ALLDAY, see https://www.e-dynamics.be/wordpress/conditional-tags/

**Comment by dailysol** [comment]
Thank you for getting back to me quickly.
We don't want designated times on the calendar.  It is ongoing, example every Monday.  
We do not want to give hours.  We want them to contact us because there are a lot of variations and set a time for the day. 
Do we use the "custom field"  to hide the time and still have the recurrence.  I would need the code to hide the time.

**Comment by liedekef** [comment]
in the settings, you can change the layout of individual events in the calendar too, so change those to your liking.

**Reply by dailysol** [reply]
What do I need to put in the custom field so there will not showing times in certain recurrence events?

**Reply by liedekef** [reply]
you don't need to use custom fields, you need to use eme_if , see the link in my earlier comment.

**Reply by dailysol** [reply]
Below is the closest for putting it together like this although that is the date that I can see am I closer??  It would be:: #H:#i
[eme_if tag="#ESC_{j M Y}" notvalue="#ESC@_{j M Y}"] This date is only printed if different from start date: #@_{- j M Y} [/eme_if]

**Reply by liedekef** [reply]
if you don't want to show the time for all day events: `[eme_if tag="#_IS_ALLDAY" value="0"]#H:#i[/eme_if]` , the example you quoted will compare the start and end dates and only show the end date if different from the startdate. It depends on what you want :-)

**Comment by dailysol** [comment]
I am so sorry.. I am creating the event like I usually do and I am in the event description. 
Where does this go?  [eme_if tag="#_IS_ALLDAY" value="0"]#H:#i[/eme_if]

**Comment by liedekef** [comment, marked as answer]
I think you need to check out the video first: https://www.youtube.com/watch?v=TCXQbgsrSZA
After that, change the single event format to your liking: either in the generic EME settings, or per event via a template or direct override:
![image](https://github.com/liedekef/events-made-easy/assets/75667/881a4bf6-0c43-45e2-b370-07b7edd1955f)

**Comment by ebussinetz** [comment]
How can I change this: [eme_if tag="#ESC_STARTDATE" value="#ESC_ENDDATE"]- #_ENDDATE [/eme_if]
If the ENDDATE is NOT like the STARTDATE, display the Enddate.
I would display here only the ENDDATE if needed.
Thank you!

**Reply by liedekef** [reply]
See https://www.e-dynamics.be/wordpress/conditional-tags/ , use either "notvalue" or "ne" ("ne" being the abbreviation for "not equal"):
`[eme_if tag="#ESC_STARTDATE" notvalue="#ESC_ENDDATE"]- #_ENDDATE [/eme_if]`

**Comment by ebussinetz** [comment]
Perfect, thank you!
